test/tutorial/test_README_example.cpp

```diff
@@ -38,13 +38,19 @@ int main(const int argc, const char *argv[])
   state.conditions_[0].pressure_ = 101319.9;   // Pa
   state.SetConcentration(foo, 20.0);           // mol m-3
 
-  std::cout << "foo, bar, baz" << std::endl;
+  std::cout << std::setw(5) << "time," 
+            << std::setw(13) << "foo, "
+            << std::setw(12) << "bar, "
+            << std::setw(10) << "baz" << std::endl;
   for (int i = 0; i < 10; ++i)
   {
     auto result = solver.Solve(500.0, state);
     state.variables_ = result.result_;
-    std::cout << std::fixed << std::setprecision(6) << state.variables_[0][state.variable_map_["Foo"]] << ", "
-              << state.variables_[0][state.variable_map_["Bar"]] << ", " << state.variables_[0][state.variable_map_["Baz"]]
+    std::cout << std::fixed << std::setprecision(6) 
+              << std::setw(4) << i << ", "
```

Comment:
```suggestion
              << std::setw(8) << i * 500.0 << ", "
```
just because the timestep passed to the solver is 500.0 s

Reply:
done